app/models/nta_upload.rb

```diff
@@ -0,0 +1,5 @@
+class NtaUpload < ActiveRecord::Base
+
+  ### Implementation in another branch
```

Comment:
I have done something astoundingly dumb. Will address comments and let you know...